mrtt-ui/src/components/SiteInterventions.js

```diff
@@ -152,6 +207,55 @@ function SiteInterventionsForm() {
   const getBiophysicalIntervention = (intervention) =>
     biophysicalInterventionsFields.find((field) => field.interventionType === intervention)
 
+  const handleMangroveSpeciesUsedOnChange = (event, specie) => {
+    const mangroveSpeciesUsedCheckedCopy = mangroveSpeciesUsedChecked
```

Comment:
I dont think this is a real copy, its just a reference to the original I believe. 


This creates a shallow copy. 
`const copy = [...original]`

Reply:
oops, that one slipped in there. will update